libcontainerd/client_daemon.go

```diff
@@ -563,7 +563,8 @@ func (c *client) CreateCheckpoint(ctx context.Context, containerID, checkpointDi
 
 	opts := []containerd.CheckpointTaskOpts{}
 	if exit {
-		opts = append(opts, containerd.WithExit)
+		// Revendoring hack: please disregard.
+		//opts = append(opts, containerd.WithExit)
```

Comment:
🤔

Reply:
As it says on the tin, sorry for the revendoring noise :)